### DISCUSSION TITLE
Calendar sync

### DISCUSSION THREAD

**Discussion by GordianDziwis** [kimai/kimai#2985] [Questions & Answers] [answered]
There are some issues and source files relating to a google calendar integration. But I cannot find a menu in kimai2 relating to this.

Is this feature deprecated?

**Comment by kevinpapst** [comment, marked as answer]
https://www.kimai.org/documentation/local-yaml.html#integrating-google-calender

**Reply by GordianDziwis** [reply]
thx

**Reply by kevinpapst** [reply]
Just wanted to check if the docs are sufficient, seems they are 😁

**Reply by GordianDziwis** [reply]
They are sufficient, but I did not find it. And I really searched even searched the source code for the feature.

**Reply by GordianDziwis** [reply]
I would guess it is for self-hosting only.

**Reply by kevinpapst** [reply]
Should add a link from the calendar docs,  probably I removed them by accident here:
https://github.com/kimai/www.kimai.org/blob/master/_documentation/calendar.md
